.github/workflows/javascript-code-analysis.yml

```diff
@@ -0,0 +1,38 @@
+name: Javascript
+
+on:
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - main
+
+jobs:
+    javascript-code-analysis:
```

Comment:
this should be in the reusable workflow, see https://shopware.atlassian.net/browse/PT-13120